evaluation.md

```diff
@@ -2,18 +2,47 @@
 
 ## Internal DSL
 
-_Describe each change from your ideal syntax to the syntax you implemented, and
-describe_ why _you made the change._
+**Describe each change from your ideal syntax to the syntax you implemented, and
+describe_ why _you made the change.**
+
+First, we changed "def" to "state", because def is a reserved keyword in scala, so we couldn't use it.
+
+Next, we got rid of colons after the state name. We tried to make these into a function, but we decided to just use the apply function.
+
+We also had to change the "while" and "if" keywords to "loop" and "cond", since these were also reserved.
+
+We then realized that scala need a TON of parentheses in order to figure out what was going on.... so our code looks like lisp.
```

Comment:
I get the impression that you've used quite a few curried functions here and that that was mostly how you passed multiple parameters, and I think that contributed to the parentheses. My approach to the internal DSL probably wouldn't work so well for you two, though, since I mostly passed a bunch of parameters to just a few functions, which worked out nicely since my ideal syntax had commas in it anyway.